contract/views/contract.xml

```diff
@@ -249,11 +249,17 @@
                                     />
                                     <field name="recurring_next_date" invisible="1" />
                                     <field name="date_start" invisible="1" />
-                                    <field name="date_end" />
+                                    <field name="date_end" invisible="1" />
```

Comment:
Why making it invisible?

Reply:
Well, I make it invisible since the other fields like date_start or recurring_next_date are invisible. To be uniform, but I'm letting it visible.